package.json

```diff
@@ -10,7 +10,7 @@
     "src"
   ],
   "dependencies": {
-    "php-parser": "glayzzle/php-parser#82233e9f93a7525ed884bba65fb0369de12b0ab4",
+    "php-parser": "czosel/php-parser#0590fd4c129ddb1298a8639e69f134afb5977d20",
```

Comment:
Let's wait when `php-parser` merge you PR and we can merge this PR :+1:

Reply:
:+1: